src/core/src/main/js/dom/DomQuery.js

```diff
@@ -32,1552 +32,9 @@
-            result = result.reverse();
-          }
-        }
-
-        result = DomQuery(result);
-
-        if (filter) {
-          return result.filter(filter);
-        }
-
-        return result;
-      };
-    });
-
-    /**
-     * Returns true/false if the current set items matches the selector.
-     *
-     * @method is
-     * @param {String} selector Selector to match the elements against.
-     * @return {Boolean} True/false if the current set matches the selector.
-     */
-    DomQuery.fn.is = function (selector) {
-      return !!selector && this.filter(selector).length > 0;
-    };
-
-    DomQuery.fn.init.prototype = DomQuery.fn;
-
-    DomQuery.overrideDefaults = function (callback) {
-      var defaults;
-
-      var sub = function (selector, context) {
-        defaults = defaults || callback();
-
-        if (arguments.length === 0) {
-          selector = defaults.element;
-        }
-
-        if (!context) {
-          context = defaults.context;
-        }
-
-        return new sub.fn.init(selector, context);
-      };
-
-      DomQuery.extend(sub, this);
-
-      return sub;
-    };
-
-    var appendHooks = function (targetHooks, prop, hooks) {
-      each(hooks, function (name, func) {
-        targetHooks[name] = targetHooks[name] || {};
-        targetHooks[name][prop] = func;
-      });
-    };
-
-    if (Env.ie && Env.ie < 8) {
-      appendHooks(attrHooks, 'get', {
-        maxlength: function (elm) {
-          var value = elm.maxLength;
-
-          if (value === 0x7fffffff) {
-            return undef;
-          }
-
-          return value;
-        },
-
-        size: function (elm) {
-          var value = elm.size;
-
-          if (value === 20) {
-            return undef;
-          }
-
-          return value;
-        },
-
-        'class': function (elm) {
-          return elm.className;
-        },
-
-        style: function (elm) {
-          var value = elm.style.cssText;
-
-          if (value.length === 0) {
-            return undef;
-          }
-
-          return value;
-        }
-      });
-
-      appendHooks(attrHooks, 'set', {
-        'class': function (elm, value) {
-          elm.className = value;
-        },
-
-        style: function (elm, value) {
-          elm.style.cssText = value;
-        }
-      });
-    }
-
-    if (Env.ie && Env.ie < 9) {
-      /*jshint sub:true */
-      /*eslint dot-notation: 0*/
-      cssFix['float'] = 'styleFloat';
-
-      appendHooks(cssHooks, 'set', {
-        opacity: function (elm, value) {
-          var style = elm.style;
-
-          if (value === null || value === '') {
-            style.removeAttribute('filter');
-          } else {
-            style.zoom = 1;
-            style.filter = 'alpha(opacity=' + (value * 100) + ')';
-          }
-        }
-      });
-    }
-
-    DomQuery.attrHooks = attrHooks;
-    DomQuery.cssHooks = cssHooks;
-
-    return DomQuery;
+  function (window) {
+    return window.JQuery;
```

Comment:
I think you want **lowercase** j for `jQuery`

Reply:
:(